static/js/index.js

```diff
@@ -1,6 +1,16 @@
 function renderGitHubCommits() {
   let updated = "";
   latestCommits.forEach((commit) => {
+    // first create a date object
+    const date = new Date(commit["date"]+" UTC");
```

Comment:
adding "UTC" won't be necessary if you use `.isoformat` in app.py.

Reply:
no, adding UTC makes it so that tolocaledatestring returns local time, that doesnt happen without adding UTC at the end of the string